addons/im_livechat/static/src/new/chat_window/chat_window_patch.xml

```diff
@@ -10,5 +10,19 @@
             />
            <t t-else="">$0</t>
         </xpath>
+        <xpath expr="//Composer" position="replace">
+            <t t-if="chatbotService.inputEnabled">$0</t>
+            <t t-else="">
+                <span class="bg-200 py-1 text-center fst-italic" t-esc="chatbotService.inputDisabledText"/>
+            </t>
+        </xpath>
+        <xpath expr="//i[hasclass('fa-close')]/ancestor::div[hasclass('o-mail-ChatWindow-command')]" position="before">
+            <div t-if="chatbotService.completed and !chatbotService.currentStep?.operatorFound" class="o-mail-ChatWindow-command d-flex align-items-center h-100 px-3 opacity-50 opacity-100-hover" title="Restart Conversation" t-on-click.stop="() => chatbotService.restart()">
+                <i class="fa fa-refresh"/>
+            </div>
+        </xpath>
+        <xpath expr="//*[@t-name='unreadCounter']" position="replace">
+            <t t-if="!chatbotService.active">$0</t>
+        </xpath>
```

Comment:
It's more natural to place the different xpath by order in template. Usually it matches visually from top to bottom, and from left to right. I would expect xpatch on header before xpatch on composer.